docs/tutoriales/guia-github.md

```diff
@@ -0,0 +1,187 @@
+---
+id: guia-github
+title: Guía GitHub
+sidebar_label: Guía GitHub
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+
+[GitHub](https://docs.github.com/en/free-pro-team@latest/github) es un sistema de gestión de proyectos en la nube que permite a los desarrolladores almacenar y tener un control de versiones de código. Esta plataforma te permite trabajar en colaboración con otras personas alrededor del mundo, realizar seguimientos de trabajos y administrar proyectos.  
+[Conoce más de GitHub](https://www.xataka.com/basics/que-github-que-que-le-ofrece-a-desarrolladores)  
+
+Para EOS Costa Rica, es de suma importancia que nuestros colaboradores se familiaricen con esta herramienta, por lo que les ofrecemos a continuación una guía base para empezar en el mundo de GitHub.
+
+* * *
+## **Herramientas**
+
+### [GitHub CLI](https://docs.github.com/es/free-pro-team@latest/github/getting-started-with-github/github-cli)
+
+[CLI](https://cli.github.com) de GitHub es una herramienta de línea de comandos para utilizar la funciones de GitHub en su propia computadora. 
+
+<img alt="Imagen GitHub CLI" src={ useBaseUrl( 'img/LogosGitHub/GitHub-CLI.png' )} />
+
+### [GitHub Desktop](https://docs.github.com/es/free-pro-team@latest/github/getting-started-with-github/github-desktop)
+
+[GitHub Desktop](https://docs.github.com/es/free-pro-team@latest/desktop/installing-and-configuring-github-desktop/installing-github-desktop) es la aplicación de escritorio que permite interactuar al usuario con las funcionalidades de Github por medio de la interfaz gráfica, remplazando las líneas de código por un dispositivo visual.
+
+<img alt="Imagen GitHub Desktop" src={ useBaseUrl( 'img/LogosGitHub/GitHub-Desktop.png' )} />
+
+### [GitHub para móviles](https://docs.github.com/es/free-pro-team@latest/github/getting-started-with-github/github-for-mobile)
+
+GitHub para [dispositivos móviles](https://play.google.com/store/apps/details?id=com.github.android&hl=es_419&gl=US) está disponible para Android e iOS, te ofrece una forma de realizar un trabajo de alto impacto en GitHub de forma rápida y desde cualquier lugar. Además se vuelve una manera segura y estable de acceder a tus datos de GitHub a través de una aplicación cliente confiable de primera parte.
```

Comment:
@ldrojas  I'm no sure if is a good option to use the second person singular to explain this

Reply:
I implement the text written by GitHub in your Github Docs, but I don't know if it's ok @ldrojas

Reply:
What's important here is consistency. Let's stick to the third person, for instance, instead of "acceder a tus datos" use "acceder a los datos."

Reply:
Thanks!! 😄